File: Benchmarks/benchmark.py
from timeit import default_timer as timer

def fib(n):
    return 1 if n <= 2 else fib(n - 1) + fib(n - 2)

start = timer()

for i in range(1, 41):
    fib(i)

print("First 40 Fibonnaci numbers in: ", (timer()-start) * 100, "milliseconds")

File: Benchmarks/benchmark.go
package main

import (
	"fmt"
	"time"
)

func fib(n int) int {
	if n <= 2 {
		return 2
	} else {
		return fib(n-1) + fib(n-2)
	}
}

func main() {
	start := time.Now()
	for i := 0; i < 40; i++ {
		fib(i)
	}
	fmt.Println("First 40 Fibonnaci numbers in:", time.Since(start).Milliseconds(), "milliseconds.")
}
